src/main/java/org/assertj/core/api/AbstractClassAssert.java

```diff
@@ -647,4 +647,59 @@ public SELF hasPublicMethods(String... methodNames) {
     classes.assertHasPublicMethods(info, actual, methodNames);
     return myself;
   }
+
+  /**
+   * Verifies that the actual {@code Class} has the given package name.
+   * <p>
+   * Example:
+   * <pre><code>
+   * package one.two;
```

Comment:
Join lines:

```suggestion
   * <pre><code class='java'> package one.two;
```

Reply:
Ok, but doesn't this mess up the indentation? In Intellij preview it seems so.